tests/tests/test_decommission.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import pytest
+
+from ..common_setup import standard_setup_one_client
+from ..MenderAPI import authentication, devauth
+from .common_connect import wait_for_connect
+from .mendertesting import MenderTesting
+
+
+class TestDecommission(MenderTesting):
+    """Tests the decommission and re authentication functionality"""
+
+    def test_decommission(self, standard_setup_one_client):
+        mender_device = standard_setup_one_client.device
+
+        # accept the device
+        devauth.accept_devices(1)
+
+        # Check from client side that the authorization was successful
+        out = mender_device.run(
+            "journalctl -u %s -n 6" % mender_device.get_client_service_name()
+        )
+        assert "Server authorization successful" in out
+
+        # list of devices
```

Comment:
Ummm, the actual intention of this part is to "check that the device is authorized"? My reasoning: "list of devices" is quite vague, and somehow this should be the mirror logic of lines 54-55